config/proto/targets.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto2";
+
+package hermes.targets
+
+option go_package = "github.com/googleinterns/hermes/targets/proto";
+
+message TargetsDef {
+  enum TargetSystem {
+    GOOGLE_CLOUD_STORAGE = 0;
+    CEPH = 1;
+  }
+
+  enum ConnectionType {
+   HTTP = 0;
+   HTTPS = 1;
+  }
+
+  required TargetSystem target_system = 1;
+  ConnectionType connection_type = 2;
+  ConnectionDetails connection_details = 3;
+  required int64 total_space_allocated = 4;
+}
+
+message ConnectionDetails {
```

Comment:
Add comment specifying when this is needed

Reply:
```suggestion
// Necessary when connecting to an API (e.g. Ceph).
// Unclear yet if we need this for GCS.
message ConnectionDetails {
```